### PR TITLE
Add the _Exit symbol

### PR DESCRIPTION
**Pull request by tomaka**
According to [this link](http://pubs.opengroup.org/onlinepubs/000095399/functions/exit.html) the `_Exit` symbol is a valid alias for `exit`.
This PR adds it.

### CODE REVIEW

File: src/library.js [hunk 1]
```diff
@@ -494,6 +494,10 @@ LibraryManager.library = {
   exit: function(status) {
     __exit(status);
   },
+  _Exit__deps: ['_exit'],
```
Comment (kripken) [inline src/library.js]: this can just be `_Exit: 'exit'`, and the library system will map it.

**Comment by kripken** [comment]
Please add yourself to AUTHORS.

Also we should test this. Simplest is probably to make `tests/test_core.py`'s `test_exit_status` check both `exit` and `_Exit`.

**Comment by tomaka** [comment]
Done

**Comment by kripken** [comment]
Thanks!

**Pull request merged**